ipc/api/src/evm.rs

```diff
@@ -232,6 +233,45 @@ impl TryFrom<SupplySource> for register_subnet_facet::SupplySource {
     }
 }
 
+impl TryFrom<ConstructParams> for register_subnet_facet::ConstructorParams {
+    type Error = anyhow::Error;
+
+    fn try_from(params: ConstructParams) -> Result<Self, Self::Error> {
+        let gateway = params
+            .ipc_gateway_addr
+            .ok_or_else(|| anyhow!("gateway is not defined"))?;
+        let ipc_gateway_addr = payload_to_evm_address(gateway.payload())?;
+
+        let min_validator_stake = params
+            .min_validator_stake
+            .atto()
+            .to_u128()
+            .ok_or_else(|| anyhow!("invalid min validator stake"))?;
+
+        let route = subnet_id_to_evm_addresses(&params.parent)?;
+        log::debug!("root SubnetID as Ethereum type: {route:?}");
```

Comment:
Do we really need to keep this debug log in a conversion function?

Reply:
I mean I can imagine it aided you during development, but does it serve an ongoing purpose for the operator interacting with the CLI?

Reply:
yeah, we can remove them, just copy pasted from previous code.